orchestra/models/core/models.py

```diff
@@ -523,3 +526,64 @@ class PayRate(PayRateMixin, models.Model):
     hourly_multiplier = models.DecimalField(max_digits=6, decimal_places=4)
     start_date = models.DateField(default=timezone.now)
     end_date = models.DateField(null=True, blank=True)
+
+
+class StaffingRequest(StaffingRequestMixin, models.Model):
```

Comment:
I think these should be in `orchestra/models/communciation/models.py` or we can make a `bots` models dir.
I think more separation of the models is better :)